eth_keys/backends/__init__.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import absolute_import
 
 import os
+from typing import Optional  # noqa: F401
```

Comment:
Assuming this is due to the use of this within the type annotation comments.  Odd but also makes sense.

Reply:
It is, yes